README.md

```diff
@@ -49,7 +49,7 @@ For more information on configuration see
 
 To configure the schema for connector to work:
 ```
-	elisctl queue change SCHEMA_ID -s example_schema.json
+	elisctl queue change 29582 SCHEMA_ID -s example_schema.json
```

Comment:
I'd maybe reference the `QUEUE_NUMBER` here since we already have it defined. Also just to be consistent with other arguments (`SCHEMA_ID`). Maybe we could even change `QUEUE_NUMBER` to `QUEUE_ID`? (as I don't think we mention queue numbers anywhere).

Note: Note how `QUEUE_NUMBER` sounds very much like `CUCUMBER`.

Reply:
I like your note!